atd-vze/src/queries/gqlAbstract.js

```diff
@@ -224,10 +244,20 @@ gqlAbstractTableAggregateName (
 
     if (this.config["where"] !== null) {
       let where = [];
+      let _or = [];
       for (let [key, value] of this.getEntries("where")) {
         where.push(`${key}: {${value}}`);
       }
-      output.push(`where: {${where.join(", ")}}`);
+      if (this.config["or"] && this.config["or"] !== null) {
```

Comment:
would the double-bang trick work here?  
`!!this.config["or"]` instead of the two conditionals? 

https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054